Ginger/GingerCoreNET/ActionsLib/Webservices/ActWebAPIModel.cs

```diff
@@ -203,6 +203,12 @@ private void FillAPIBaseFields(ApplicationAPIModel AAMB, ActWebAPIBase actWebAPI
 
             actWebAPIBase.ReturnValues = actWebAPIModel.ReturnValues;
             actWebAPIBase.Context = actWebAPIModel.Context;
+
+            Context context = Amdocs.Ginger.Common.Context.GetAsContext(actWebAPIBase.Context);
+            if (context != null && context.Runner != null)
+            {
+                context.Runner.PrepActionValueExpression(actWebAPIBase, context.BusinessFlow);
```

Comment:
let's talk on it, i want to understand:
1-  how till now this was not needed and now it is
2- if that the place to do it for making sure all VE in action were calculated

Reply:
Updated the code as per suggestions.